client/src/hooks/Chat/useFocusChatEffect.ts

```diff
@@ -11,11 +11,16 @@ export default function useFocusChatEffect(textAreaRef: React.RefObject<HTMLText
         'conversation',
         `Focusing textarea on location state change: ${location.pathname}`,
       );
+
       /** Check if the device is not a touchscreen */
       if (!window.matchMedia?.('(pointer: coarse)').matches) {
         textAreaRef.current?.focus();
       }
-      navigate(`${location.pathname}${location.search ?? ''}`, { replace: true, state: {} });
+
+      navigate(`${location.pathname}${window.location.search}`, {
+        replace: true,
+        state: {},
+      });
```

Comment:
Just to clarify what's going on here. Since I wasn't able to use the react router functions in the atom family in https://github.com/danny-avila/LibreChat/pull/7151, then it wasn't being populated into the react router functions like `const location = useLocation()` and this changes it to pull from the actual `window.location.search` instead, where `location.search` was empty and clobbering everything.